pyfr/integrators/dual/phys/base.py

```diff
@@ -49,6 +49,22 @@ def grad_soln(self):
 
         return self._curr_grad_soln
 
+    @property
+    def dt_soln(self):
```

Comment:
Can we try and factor this method out across dual time and std? (Same goes for its friends above.)